Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -249,6 +249,13 @@ final class PurchasesOrchestrator {
                 return
             }
 
+            if !self.receiptParser.receiptHasTransactions(receiptData: receiptData) {
```

Comment:
😇 our integration tests caught that this is now running in the main thread, which is bad for performance
```
0   libswiftCore.dylib            	       0x10fe98e81 _assertionFailure(_:_:file:line:flags:) + 353
1   RevenueCat                    	       0x1402cee1a static PurchasesReceiptParser.ensureRunningOutsideOfMainThread() + 842 (PurchasesReceiptParser.swift:127)
2   RevenueCat                    	       0x1402ce459 PurchasesReceiptParser.parse(from:) + 217 (PurchasesReceiptParser.swift:43)
```

Reply:
Good catch tests! :D